src/Components/test/testassets/BasicTestApp/InteropComponent.razor

```diff
@@ -309,6 +312,15 @@
         ReturnValues.TryAdd(testName, "Success");
     }
 
+    private async Task<string> SendDisposedStreamToJSAsync(bool shouldRead)
+    {
+        var stream = DotNetStreamReferenceInterop.GetDotNetStreamReference();
+        var task = JSRuntime.InvokeAsync<string>("jsInteropTests.receiveDisposedDotNetStreamReference", stream, shouldRead);
+        stream.Dispose();
+
+        return await task;
```

Comment:
If I understand correctly, this was throwing an error early on in the .NET side that was causing the circuit to get terminated and now the error will surface on the JS side?

Reply:
Right, except it's more that the stream got disposed before the client was able to initiate the call to the `SendDotNetStreamToJS` hub method.